src/sentry/static/sentry/app/styles/input.jsx

```diff
@@ -38,6 +38,13 @@ const inputStyles = props => {
 
     &[disabled] {
       background: ${props.theme.whiteDark};
+      color: ${props.theme.gray2};
+      border: 1px solid ${props.theme.borderDark};
+      cursor: not-allowed;
+
+      &::placeholder {
+        color: ${props.theme.gray1};
+      }
```

Comment:
Do we want radio buttons to match these colours? Right now they have their [own styles that use opacity](https://github.com/getsentry/sentry/blob/0405224a6f38e50223fa9ca8072d7db03ca508ce/src/sentry/static/sentry/app/views/settings/components/forms/controls/radioGroup.jsx#L56-L82). Select boxes [use theme.disabled](https://github.com/getsentry/sentry/blob/0405224a6f38e50223fa9ca8072d7db03ca508ce/src/sentry/static/sentry/app/components/forms/selectControl.jsx#L116-L118). Should we use that here for the `color` and potentially update it to `theme.gray2`?

Reply:
Good idea! Though I believe the current value we have for theme.disabled is slightly too light. I'll update it and use it here instead.